### PR TITLE
Add 0.3 version of tutorial to work with current interpreter

### PR DESCRIPTION
**Pull request by pyrocto**
Add 0.3 version of tutorial to work with current interpreter

### CODE REVIEW

**Comment by pyrocto** [comment]
Everything with line numbers has been checked to run on rchain.cloud.

File: docs/rholang/rholangtut-0.3.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
+# A Rholang tutorial
+
+Rholang is a new programming language designed for use in distributed systems.  Like all newborn things, it is growing and changing rapidly; this document describes the syntax that will be used in the RNode-0.2 release.
```
Comment (birchmd) [inline docs/rholang/rholangtut-0.3.md]: Update RNode version number

**Comment by henrytill** [comment]
🤘 🤘 🤘

**Comment by henrytill** [comment]
maiden

**Pull request merged**